bin/aerys

```diff
@@ -111,12 +105,17 @@ $climate->arguments->add([
 ]);
 
 $console = new Aerys\Console($climate);
+
 try {
     if ($console->isArgDefined("help")) {
         echo $help;
         exit(0);
     }
-} catch (Exception $e) {
+
+    if (!$console->isArgDefined("config")) {
+        throw new \Error("A config file must be defined using -c or --config");
```

Comment:
Why this manual check?

Reply:
Running `bin/aerys --help` would say the command arguments were invalid. To fix that I had to remove the requirement for config, but then manually check for it if help wasn't requested.